recipes/httpstan/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "httpstan" %}
+{% set version = "4.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stan-dev/httpstan/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f4d81e2ecbf17fce6a03fea7200d10ab03fbad1a7f3db1453464ea12c4fa740c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python  >=3.7
+    - pybind11  >=2.6.2
+    - poetry  >=1.1.4
+  run:
+    - python  >=3.7
```

Comment:
Looks like additional run dependencies like `numpy` and `aiohttp` listed [here](https://github.com/stan-dev/httpstan/blob/2425e2d443e938fe35c5a4b8c36f3b1e5c00f259/pyproject.toml#L33-L40) and appearing in the codebase are missing.